packages/core/src/tw/component/components/paragraph.ts

```diff
@@ -44,8 +44,15 @@ export class ParagraphModelNode extends BlockModelNode<IParagraphAttributes> {
     }
 }
 
-export interface IParagraphStyle extends IStyle {}
+export interface IParagraphStyle extends IStyle {
+    textAlign: 'left' | 'right' | 'center'
+    direction: 'rtl' | 'ltr'
```

Comment:
These must also be defined in the attributes interface on line 16. See `tw/component/components/text.ts` for example.

As for lint, this project uses semi-colons and trailing commas (line 54). I'll hook up a bot later to auto comment on lint.